### DISCUSSION TITLE
Possibility to remove some css units from properties

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#4464] [Ideas]
Hello Arthur.
We have to remove some css properties from the style manager this way: 
```js
// After the init without any css sector, we update some builtIn with the units needed:
        const sm = editor.StyleManager;
        sm.addBuiltIn('width', {
            units: ['%', 'px']
        });
        sm.addBuiltIn('height', {
            units: ['%', 'px']
        });
// And re-add the sections like it: 
        editor.StyleManager.addSector('dimension', {
            name: 'Dimension',
            open: false,
            buildProps: ['width', 'height', 'min-height', 'margin', 'padding'],
        })
```
So, we could have some perperty to change these on init in grapes to change the css properties?
Or, one other way i can figure out is set the `value="em"` in the options so we can hide with css.
![image](https://user-images.githubusercontent.com/2287371/179999065-867dfb24-597d-4922-80b8-6606f29e8adb.png)

**Comment by artf** [comment]
If you're configuring your style manager manually from init you can specify there your units for a specific property (eg. https://grapesjs.com/docs/modules/Style-manager.html#property-defintions) otherwise is probably better what you're doing right now as you don't need to redeclare all other properties.

**Reply by ronaldohoch** [reply]
Hmmm, i really forgot about that part of documentation :(

Thank you Artf.
